src/librustc_driver/driver.rs

```diff
@@ -243,7 +248,48 @@ pub fn compile_input(sess: &Session,
                 tcx.print_debug_stats();
             }
 
-            let trans = phase_4_translate_to_llvm(tcx, analysis, incremental_hashes_map,
+            #[cfg(not(feature="llvm"))]
```

Comment:
Consider moving this outside of the main driver function?

Reply:
Should remove the #[cfg]